devtools/conda-envs/test_env.yaml

```diff
@@ -17,7 +17,7 @@ dependencies:
   - numexpr
   - autograd
   - pymbar
-  - openmm=7.5.1
+  - openmm=7.6
```

Comment:
I think we don't want to pin any specific `openmm` version. It should use the latest available and it would also not require manually changing it for future openmm releases. Isn't that the case?

Reply:
I don't think there is an issue with having to manually change as openmm releases come out since if a new version comes out that breaks things, then we would want to keep testing against the older version while we update.

Reply:
I'm open to other ideas! Also pinning the "oldest" version we support makes the logic of refining the env a bit easier in the CI, but we could probably refactor it so we don't need to do that.

Reply:
We should pin `openmm >=7.6`, not `=`!